core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/BaseResourceUpdateRequest.java

```diff
@@ -28,17 +28,17 @@
 
 	private static final long serialVersionUID = 1L;
 
-	protected String url;
-	protected String title;
-	protected String language;
-	protected String description;
-	protected String status;
-	protected String copyright;
-	protected String owner;
-	protected String contact;
-	protected String usage;
-	protected String purpose;
-	protected String bundleId;
+	String url;
```

Comment:
Fields should be private here as well, if you need access then create a protected accessor for it.

Reply:
Also, add `JsonProperty` to all fields so we get them in the audit logs. Applies to the other requests as well.